src/Sentry/Laravel/EventHandler.php

```diff
@@ -115,7 +115,7 @@ public function subscribeQueueEvents(QueueManager $queue)
             Integration::flushEvents();
 
             // We have added a scope when a job starts processing
-            Hub::getCurrent()->popScope();
+            SentrySdk::getCurrentHub()->popScope();
```

Comment:
This approach will require contraining `sentry-php` to `^2.2`; I thought @HazAT preferred the code-based approach, did I got it wrong?

Reply:
`code-based approach`? What did I miss?

Reply:
The code-based approach would be using `if (class_exists(SentrySdk::class)) { // ... }` or something equivalent.

Reply:
I think we need to do it since we didn't lock the version to minor meaning it could happen that someone updates the core without laravel, so I would prefer the gatekeeper
`if (class_exists(SentrySdk::class)) {` with a comment to remove at some point in the future.

Reply:
Hmm. Good point. Okay 👌

Reply:
Updated to do a class exists check with plenty of comments about it's removal.